lib/ansible/modules/files/find.py

```diff
@@ -345,6 +365,9 @@ def main():
 
     params = module.params
 
+    if params['contains'] and not params['contains_anywhere']:
+        module.deprecate("contains will default to using contains_anywhere as yes in future.", version='3.0')
```

Comment:
The deprecation version will be 2.11, not 3.0

Reply:
Use True instead of yes just to avoid ambiguity. YAML types are rather fuzzy, True and False are less ambiguous.